app/adapters/alma_adapter/alma_item.rb

```diff
@@ -70,11 +70,26 @@ def enum_cron
     end
 
     def enumeration
-      item.item_data["enumeration_a"]
+      enums = []
```

Comment:
Mark told us that enum_a is populated during the migration from voyager, but that we'll start using the others. I don't think we have a fixture supporting this yet.

Reply:
Great, resolving.

Reply:
I have seen records with at least two of them populated (enum_a and enum_b I believe). But it made sense to add them all since people might start using the rest once Alma is live.